osu.Game/Screens/OnlinePlay/Lounge/DrawableLoungeRoom.cs

```diff
@@ -122,7 +122,7 @@ public bool MatchingFilter
             }
         }
 
-        public Popover GetPopover() => new PasswordEntryPopover(Room) { JoinRequested = lounge.Join };
+        public Popover GetPopover() => new PasswordEntryPopover(Room) { Lounge = lounge };
```

Comment:
Components aren't usually passed down manually like this. Resolving locally inside the popover (the same way `DrawableLoungeRoom` does) is preferable and should work as well.

Reply:
Updated PR with requested changes.